contents.txt

```diff
@@ -3848,7 +2985,6 @@
 ./shared/Microsoft.NETCore.App/6.0.0/System.Xml.XmlSerializer.dll
 ./shared/Microsoft.NETCore.App/6.0.0/System.Xml.XPath.dll
 ./shared/Microsoft.NETCore.App/6.0.0/System.Xml.XPath.XDocument.dll
-./shared/Microsoft.NETCore.App/6.0.0/.version
```

Comment:
> [@omajid]
> I can't see anything that uses the runtime's .version file (though I do see some users of the SDK's version file). Still, this seems like something that might break callers?

Reply:
IIRC this file is intended to be purely informational--it would contain the commit hash on the first line and then the version number on a second line. IMO would be good to put back in place for parity, although I doubt it would cause problems with "ordinary" SDK/sharedfx usage.